bootstrap/cmd/kfdefConverter/cmd/tov1beta1.go

```diff
@@ -40,6 +41,8 @@ var tov1beta1Cmd = &cobra.Command{
 		if outPath == "" {
 			outPath = args[0]
 		}
+		config.Spec.AppDir = filepath.Dir(outPath)
+		config.Spec.ConfigFileName = filepath.Base(outPath)
```

Comment:
Why not store the full  path to config.Spec.ConfigFileName as opposed to just the relative path?

Reply:
Because appDir is the directory.
Or we can change appDir + configfilename to be appFile (full path)?